pkg/proxy/util/serviceimport.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	mcsv1alpha1 "k8s.io/mcs-api/pkg/apis/multicluster/v1alpha1"
+)
+
+const (
+	// ServiceImportPrefix is used to identify imported services
+	// and separate them from traditional clusters local services
+	// that may have the same name. This prefix intentionally
+	// makes a service name invalid to guarantee that there are
+	// never collisions with same-named cluster local services.
+	// The new service name may be used in log messages and other
+	// user-facing strings.
+	ServiceImportPrefix = "import:"
```

Comment:
Since there's no length limitations here, I'm wondering if it would be worthwhile to prefix this with the entire multi-cluster APIversion: `multicluster.k8s.io/v1alpha1:`

Reply:
I like that, will make that change